pygmt/datasets/load_remote_dataset.py

```diff
@@ -409,15 +428,16 @@ def _load_remote_dataset(
             f"'region' is required for {dataset.description} resolution '{resolution}'."
         )
 
-    # Currently, only grids are supported. Will support images in the future.
-    kwdict = {"T": "g", "R": region}  # region can be None
+    kwdict = {"R": region}  # region can be None
+    kind = "image" if name in {"earth_day"} else "grid"
+    kwdict["T"] = "i" if kind == "image" else "g"
```

Comment:
Is this more readable?
```suggestion
    kind = "image" if name in {"earth_day"} else "grid"
    kwdict = {
        "R": region,  # region can be None
        "T": "i" if kind == "image" else "g",
    }  
```

Reply:
Yes, good to only declare kwdict once!